### PR TITLE
add install logs

### PR DESCRIPTION
**Pull request by duartecoelhomovai**

### CODE REVIEW

**Review by diasdm — CHANGES_REQUESTED** [review]
Won't this generate multiple artifacts with the same name? Consider using the existing artifact to store installation logs too

File: .github/workflows/integration-build-platform.yml [hunk 1]
```diff
@@ -288,6 +288,22 @@ jobs:
           echo "movai_user=${MOVAI_USER}" >> $GITHUB_OUTPUT
           echo "movai_pwd=${MOVAI_PWD}" >> $GITHUB_OUTPUT
 
+      - name: Collect Install logs
+        run: |
+          # cleanup
+          rm -rf install_logs
+
+          mkdir -p install_logs
+          journalctl -u movai-service -t mobros -n 100 > install_logs/spawner-firmware.log
```
Comment (AlexFernandes-MOVAI) [inline .github/workflows/integration-build-platform.yml]: use something like `journalctl -u movai-service --since '1hour ago' &> "movai-service.log"`

**Review by AlexFernandes-MOVAI — CHANGES_REQUESTED** [review]

**Comment by diasdm** [comment]
This was looking familiar... We added movai-service logs but only on v1: https://github.com/MOV-AI/.github/pull/143

**Comment by diasdm** [comment]
I've open a PR cherry-picking what we have done before, this way all tests save the same logs in the same manner, see #162

**Comment by duartecoelhomovai** [comment]
> I've open a PR cherry-picking what we have done before, this way all tests save the same logs in the same manner, see #162

Adapted to it

**Review by AlexFernandes-MOVAI — APPROVED** [review]

File: .github/workflows/integration-build-platform.yml [hunk 2]
```diff
@@ -1504,6 +1518,23 @@ jobs:
         env:
           SIMULATION_ID: "CI"
 
+      - name: Collect Install logs
```
Comment (diasdm) [inline .github/workflows/integration-build-platform.yml]: I think these changes can be deleted, we have the logs 2 steps down

**Review by diasdm — CHANGES_REQUESTED** [review]

File: .github/workflows/integration-build-product.yml [hunk 3]
```diff
@@ -674,6 +692,24 @@ jobs:
...
+          mkdir -p install_logs
+          journalctl -u movai-service -t mobros -n 100 > install_logs/spawner-firmware.log
+          journalctl -u movai-service > install_logs/movai-service.log
+
+      - name: Stash Install logs artifacts
+        continue-on-error: true
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: install_logs
```
Comment (diasdm) [inline .github/workflows/integration-build-product.yml]: It should be different no? Not sure what happens if they are the same
```suggestion
          name: install_simulator_logs
```

**Review by diasdm — CHANGES_REQUESTED** [review]

**Review by diasdm — APPROVED** [review]

**Pull request merged**